virtualization/windowscontainers/quick-start/quick-start-windows-10-linux.md

```diff
@@ -28,7 +28,8 @@ This quick start is specific to Windows 10. Additional quick start documentation
 ## Prerequisites
 
 Please make sure you meet the following requirements:
-- One physical computer system running Windows 10 Professional or Enterprise with Fall Creators Update (version 1709) or later
```

Comment:
Maybe the Windows 10 Pro and Enterprise should also have notice about the minimum update 1809, I guess some companies still running Windows 10 1709 or 1803.

Reply:
Isn't that what I have? 1809 applies to all.